spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `sample` namespace.
+
+The `sample.customers` table manages customers' information.
+The `credit_limit` is the maximum amount of money a lender will allow each customer to spend using a credit card, and the `credit_total` is the amount of money that each customer has already spent by using the credit card.
+
+The `sample.orders` table manages orders' information, and the `sample.statements` table manages the statements' information of the orders. Finally, the `sample.items` table manages items' information to be ordered.
+
+
+The ER diagram for the schema is as follows:
+
+![ERD](images/ERD.png)
+
+### Transactions
+
+The following five transactions are implemented in this sample application:
+
+1. Getting customer information
+3. Placing an order. An order is paid by a credit card. It first checks if the amount of the money of the order exceeds the credit limit. If the check passes, it records order histories and updates the `credit_total`
+4. Getting order information by order ID
+5. Getting order information by customer ID
+6. Repayment. It reduces the amount of `credit_total`.
+
+### Configuration
+
+The configurations for the sample Spring Boot application is as follows:
+
+```application.properties
+spring.datasource.driver-class-name=com.scalar.db.sql.jdbc.SqlJdbcDriver
+spring.datasource.url=jdbc:scalardb:
+?scalar.db.storage=cassandra\
+&scalar.db.contact_points=localhost\
+&scalar.db.username=cassandra\
+&scalar.db.password=cassandra
+```
+
+Since you use Cassandra in this sample application as mentioned above, you need to configure Cassandra settings in the configuration.
+
+## Set up
+
+You need to run the following `docker-compose` command:
+```shell
+$ docker-compose up -d
+```
+
+This command starts Cassandra and loads the schema.
+Please note that you need to wait around more than one minute for the containers to be fully started.
+
+### Initial data
+
+You first need to load initial data with the following command:
+```shell
+$ ./gradlew run --args="LoadInitialData"
+```
+
+And the following data will be loaded:
+
+- For the `sample.customers` table:
+
+| customer_id | name          | credit_limit | credit_total |
+|-------------|---------------|--------------|--------------|
+| 1           | Yamada Taro   | 10000        | 0            |
+| 2           | Yamada Hanako | 10000        | 0            |
+| 3           | Suzuki Ichiro | 10000        | 0            |
+
+- For the `sample.items` table:
+
+| item_id | name   | price |
+|---------|--------|-------|
+| 1       | Apple  | 1000  |
+| 2       | Orange | 2000  |
+| 3       | Grape  | 2500  |
+| 4       | Mango  | 5000  |
+| 5       | Melon  | 3000  |
+
+## Run the sample application
+
+Let's start with getting the customer information whose ID is `1`:
+```shell
```

Comment:
Let's add a paragraph break before the code block since static site generators may not render it properly without the break.

```suggestion
Let's start with getting information about the customer whose ID is `1`:

```shell
```